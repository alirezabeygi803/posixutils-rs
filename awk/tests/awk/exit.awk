BEGIN {
    print "begin1";
}

BEGIN {
    print "begin2";
}

{ print "first record"; exit; }

END {
    print "end1";
}

END {
    print "end2";
}