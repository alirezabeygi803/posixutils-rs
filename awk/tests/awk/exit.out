begin1
begin2
first record
end1
end2
